Emby.AutoOrganize/Configuration/fileorganizer.js

```diff
@@ -328,8 +333,8 @@
                 dlg.querySelector('#divEpisodeChoice').classList.add('hide');
                 break;
             case "Movie":
-                dlg.querySelector('#selectMedias').setAttribute('label', 'Movie');
-                dlg.querySelector('[for="selectMedias"]').innerHTML =  'Movie';
+                mediaSelector.setAttribute('label', 'Movie');
+                if (mediaSelector && mediaSelector.setLabel) mediaSelector.setLabel('Movie');
```

Comment:
When the dialog first opens, the `<label>` element for the selector has not yet been created so we check for that here. The label text is still initialized correctly by the `emby-select` component when it loads using the `label` attribute value. See: https://github.com/jellyfin/jellyfin-web/blob/master/src/components/emby-select/emby-select.js#L151